src/test/java/org/ua/javaPro/berezhnoy/hillelHomeWork12/ArrayListGenericsTest.java

```diff
@@ -0,0 +1,58 @@
+package org.ua.javaPro.berezhnoy.hillelHomeWork12;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ArrayListGenericsTest {
+    ArrayListGenerics<Object> list = new ArrayListGenerics<>();
```

Comment:
```suggestion
    ArrayListGenerics<String> list = new ArrayListGenerics<>();
```
Generics це як раз щоб не працювати з Object.